{"name": "SUC-T334 - Consultas - Saldos de caja", "status": "failed", "statusDetails": {"message": "Failed: Test Fail\nTraceback (most recent call last):\n  File \"C:\\ITAU_Tools\\QA_Automation\\workspace\\Fram27\\src\\proyectos\\Caja_Homo_suc50\\test\\SUC_T334.py\", line 52, in test_SUC_T334\n    self.seleccionarConsultas()\n  File \"C:\\ITAU_Tools\\QA_Automation\\workspace\\Fram27\\src\\proyectos\\Caja_Homo_suc50\\st\\stAdminInicio.py\", line 151, in seleccionarConsultas\n    self.fail_msg(msgFail)\n  File \"C:\\ITAU_Tools\\QA_Automation\\workspace\\Fram27\\src\\error_management.py\", line 13, in fail_msg\n    raise Exception(description)\nException: No se pudo seleccionar menu - consultas", "trace": "self = <proyectos.Caja_Homo_suc50.test.SUC_T334.SUC_T334 testMethod=test_SUC_T334>\n\n    def tearDown(self):\n>       fin(self)\n\nSUC_T334.py:64: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\st\\__init__.py:40: in fin\n    self.fail_test(self.error)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <proyectos.Caja_Homo_suc50.test.SUC_T334.SUC_T334 testMethod=test_SUC_T334>\nerror = 'Traceback (most recent call last):\\n  File \"C:\\\\ITAU_Tools\\\\QA_Automation\\\\workspace\\\\Fram27\\\\src\\\\proyectos\\\\Caja_Ho...ment.py\", line 13, in fail_msg\\n    raise Exception(description)\\nException: No se pudo seleccionar menu - consultas\\n'\n\n    def fail_test(self, error):\n        \"\"\"\n        Method to show a fail in the test. And write the error in the log\n        \"\"\"\n        msg = 'Test Fail'\n        self.capture_image(msg)\n        msg = '{msg}\\n{error}'.format(msg=msg, error=error)\n        self.log.error(msg)\n>       pytest.fail(msg)\nE       Failed: Test Fail\nE       Traceback (most recent call last):\nE         File \"C:\\ITAU_Tools\\QA_Automation\\workspace\\Fram27\\src\\proyectos\\Caja_Homo_suc50\\test\\SUC_T334.py\", line 52, in test_SUC_T334\nE           self.seleccionarConsultas()\nE         File \"C:\\ITAU_Tools\\QA_Automation\\workspace\\Fram27\\src\\proyectos\\Caja_Homo_suc50\\st\\stAdminInicio.py\", line 151, in seleccionarConsultas\nE           self.fail_msg(msgFail)\nE         File \"C:\\ITAU_Tools\\QA_Automation\\workspace\\Fram27\\src\\error_management.py\", line 13, in fail_msg\nE           raise Exception(description)\nE       Exception: No se pudo seleccionar menu - consultas\n\n..\\..\\..\\error_management.py:33: Failed"}, "description": "\n1-Ingresar a la pagina Administración de Cajas (https://fe-admincaja-dist-icaja-front-homo.ocprch.sis.ad.bia.itau/#/)</br>  \n2-Colocar los datos del Supervisor (Legajo - Terminal - Password).</br>  \nLegajo: UIC10022</br>  \nTerminal:IA0500822  </br>  \nPassword:</br>  \nHacer clic en Aceptar.</br>  \n3-Visualizar los datos del Cajero: \"Hola UIC10022 !\" </br>  \n4-Visualizar en el recuadro los datos del cajero: </br>  \nLegajo: \"UIC10022\" </br>  \nTerminal: \"IA0500822  \" </br>  \nSucursal: \"0050\" Terminal: \"IA0500822  \" Sucursal: \"0050\" </br>  \n5-Hacer clic en el Botón Consultas.</br>  \n6-Hacer clic en el Botón Saldos de Caja </br>  \n7-Visualizar el Titulo Consultas Saldos de Caja </br>  \n8-Validar que el Item Condición este \"Todos\" </br>  \n9-Validar Sucursal \"0050 - Puerto Madero\" </br>  \n10-Hacer clic en el Botón Buscar </br>  \n11-Visualizar en pantalla los saldos de caja. </br>  \n12-Hacer clic en el Boton Imprimir </br>  \n13-Hacer clic en el Boton Salir. </br>  \n   \n    ", "steps": [{"name": "PASO 0: Abrir Admin caja", "status": "passed", "attachments": [{"name": "Abrir Admin caja", "source": "f9f1e2d2-af2e-4e8d-aa5b-9f8ff4b00d1e-attachment.png", "type": "image/png"}], "start": 1699639550092, "stop": 1699639557393}, {"name": "PASO 1: Completar Usuario", "status": "passed", "attachments": [{"name": "Se pudo completar usuario", "source": "8072a203-2483-4f44-9d1a-c206c2bd2a4f-attachment.png", "type": "image/png"}], "start": 1699639560567, "stop": 1699639561851}, {"name": "PASO 2: Completar terminal", "status": "passed", "attachments": [{"name": "Se pudo completar terminal", "source": "a85a51d1-5145-4d8a-aa08-8b70a1592943-attachment.png", "type": "image/png"}], "start": 1699639561851, "stop": 1699639563191}, {"name": "PASO 3: Completar clave", "status": "passed", "attachments": [{"name": "Completar clave", "source": "42b0f096-8e1e-4a76-ab92-5c23e6768c71-attachment.png", "type": "image/png"}], "start": 1699639563191, "stop": 1699639564498}, {"name": "PASO 4: Seleccionar Ingresar", "status": "passed", "attachments": [{"name": "Se pudo seleccionar ingresar", "source": "4944a45e-63d6-4337-891b-71c791664fde-attachment.png", "type": "image/png"}], "start": 1699639564499, "stop": 1699639565978}, {"name": "PASO 5: Validar nombre del cajero", "status": "passed", "attachments": [{"name": "Validar nombre del cajero", "source": "8b80b861-1e8f-4494-a69c-e1fd7a6b4c82-attachment.png", "type": "image/png"}], "start": 1699639575986, "stop": 1699639576698}, {"name": "PASO 6: Seleccionar menu - consultas", "status": "broken", "statusDetails": {"message": "Exception: No se pudo seleccionar menu - consultas\n", "trace": "  File \"C:\\ITAU_Tools\\QA_Automation\\workspace\\Fram27\\src\\proyectos\\Caja_Homo_suc50\\st\\stAdminInicio.py\", line 151, in seleccionarConsultas\n    self.fail_msg(msgFail)\n  File \"C:\\ITAU_Tools\\QA_Automation\\workspace\\Fram27\\src\\error_management.py\", line 13, in fail_msg\n    raise Exception(description)\n"}, "attachments": [{"name": "No se pudo seleccionar menu - consultas", "source": "3e1b9ce8-e747-458f-91e3-27b96bb2aa2c-attachment.png", "type": "image/png"}], "start": 1699639581709, "stop": 1699639592982}], "attachments": [{"name": "Test Fail", "source": "5bcad05a-7b46-40b9-ad20-77cdf288c0c2-attachment.png", "type": "image/png"}, {"name": "log", "source": "51054b5e-e6ec-44d5-bd53-007f3be9cc21-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "8dfc85ba-2b81-40fe-af1f-35ba9e89cd88-attachment.txt", "type": "text/plain"}], "start": 1699639550092, "stop": 1699639594147, "uuid": "d0fb662c-2492-4686-91fd-ec80afe8a6bd", "historyId": "631cbbe25ece56dcb27f10e4cdd5e007", "fullName": "SUC_T334.SUC_T334#test_SUC_T334", "labels": [{"name": "severity", "value": "normal"}, {"name": "story", "value": "Terminales registrados"}, {"name": "feature", "value": "Consultas"}, {"name": "parentSuite"}, {"name": "suite", "value": "SUC_T334"}, {"name": "subSuite", "value": "SUC_T334"}, {"name": "host", "value": "IBA9079"}, {"name": "thread", "value": "564-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "SUC_T334"}], "links": [{"type": "test_case", "url": "Adm.Caja -SUC-T334 - Administracion de cajas - Consultas - Saldos de caja ", "name": "Adm.Caja -SUC-T334 - Administracion de cajas - Consultas - Saldos de caja "}]}